Client/Frontend/Browser/Tab.swift

```diff
@@ -43,7 +43,8 @@ struct TabState {
 
 class Tab: NSObject {
     var id: String?
-    
+    var alertShownCount: Int = 0
```

Comment:
wdyt about putting these in a struct together, since they are tied together?

Reply:
they are actually independent of each other. So keeping them as is

Reply:
I just meant since they are both related to alerts? But this is fine by me.